src/opera/storage.py

```diff
@@ -3,6 +3,12 @@
 
 
 class Storage:
+    DEFAULT_INSTANCE_PATH = ".opera"
+
+    @classmethod
+    def create(cls, instance_path: str = None) -> "Storage":
```

Comment:
This is a python quirk. If you want to add a type hint that references a class at the class level, you need to quote it, because the class name doesn't exist yet. Slightly strange that it wouldn't exist as we're obviously inside the class already, but that's the way the parser works apparently.

Reply:
Aha, interesting!